llvm/test/CodeGen/AMDGPU/wwm-regalloc-error.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: not llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 -mattr=-flat-for-global -stress-regalloc=3 -filetype=null %s 2>&1 | FileCheck %s
```

Comment:
Remove -mattr=-flat-for-global

Reply:
The attr flat-for-global was used to force SGPR usage and eventually their spills. Otherwise, it uses GLOBAL_STORE_DWORD that only need VGPRs.

Reply:
This is far too roundabout way of getting there, there must be a simpler way of getting SGPR uses. No new uses of -flat-for-global should be introduced